src/Menu.jsx

```diff
@@ -1,27 +1,21 @@
 import React from 'react';
 import ButtonMenu from './components/ButtonMenu';
 import { getAllCategories } from './api';
-import logoName from './images/bar-logo-name.png';
 
 export default function Menu() {
   const allCategories = getAllCategories();
-
   return (
-    // <DefaultLayout title={props.title} description={props.description} page="categories">
-    <div>
-      <div className="Menu">
-        <img alt="bar-logo-name" src={logoName} className="BarLogoName" />
-        {allCategories.map((categ) => (
-          <ButtonMenu
-            name={categ.title}
-            linkName="./beer-list"
-            key={categ.id}
-            data={categ.title}
-          />
-        ))}
-        <ButtonMenu name="Назад" linkName="../" />
-      </div>
-    </div>
-    // </DefaultLayout>
+    <>
+      {allCategories.map((categ) => (
+        <ButtonMenu
+          buttonProps={{
+            linkName: categ.toLowerCase(),
```

Comment:
Does router require url in lower case?

Reply:
And what about uniting of same props

Reply:
No, but i like them in lower case. :)

Reply:
ah, no, I can’t do that either, it helps to determine when display category name